test/e2e/app-dir/app/app/react-fetch/server-component/page.js

```diff
@@ -0,0 +1,18 @@
+async function getRandomMemoizedByFetch() {
+  const res = await fetch(
+    'https://next-data-api-endpoint.vercel.app/api/random'
```

Comment:
This is not deployed yet, asked @ijjk for access to the deployment so that the api route can be added.